flake.nix

```diff
@@ -172,9 +172,6 @@
                 nodePackages.eslint
                 nodePackages.prettier
                 blockfrost-backend-ryo
-                cardano-node.packages.${system}.cardano-testnet
-                cardano-node.packages.${system}.cardano-cli
```

Comment:
Why?

Reply:
Because these packages are provided as part of the runtime (`withRuntime` flag)